generated/nidmm/session.py

```diff
@@ -199,6 +203,12 @@ class _SessionBase(object):
     Specifies the powerline frequency. The NI 4050 and NI 4060 use this value to select an aperture time to reject  powerline noise by selecting the appropriate internal sample clock and filter. The NI 4065 and  NI 4070/4071/4072 use this value to select a timebase for setting the NIDMM_ATTR_APERTURE_TIME  attribute in powerline cycles (PLCs).
     After configuring powerline frequency, set the NIDMM_ATTR_APERTURE_TIME_UNITS attribute to PLCs.  When setting the NIDMM_ATTR_APERTURE_TIME attribute, select the number of PLCs for the powerline frequency.  For example, if powerline frequency = 50 Hz (or 20ms) and aperture time in PLCs = 5, then aperture time in  Seconds = 20ms * 5 PLCs = 100 ms. Similarly, if powerline frequency = 60 Hz (or 16.667 ms) and aperture time  in PLCs = 6, then aperture time in Seconds = 16.667 ms * 6 PLCs = 100 ms.
     '''
+    query_instrument_status = attributes.AttributeViBoolean(1050003)
```

Comment:
All the attributes that were removed were done so as a means to remove obsolete attributes. I added it back because it is not an obsolete attribute. 

I am not sure of the usefulness, however, if you want it removed, I can.

Reply:
This attribute has no effect on our PXI modular instruments. So it only causes confusion.